docker/seahorn-full-size-rel.Dockerfile

```diff
@@ -0,0 +1,55 @@
+#
+# Dockerfile for SeaHorn binary
+# produces package in /seahorn/build
+# Arguments:
+#  - UBUNTU:     trusty, xenial
+#  - BUILD_TYPE: debug, release
+#  - TRAVIS:     true (optional)
+#
+
+ARG UBUNTU
+
+# Pull base image.
+FROM seahorn/seahorn-build:$UBUNTU
+
+ARG TRAVIS
+
+# clone SeaHorn (just copy on Travis)
+COPY . /seahorn
+RUN if [ "$TRAVIS" != "true" ] ; \
+      then git clone https://github.com/seahorn/seahorn ./ -b master --depth=10 ; \
+    fi && \
+    mkdir -p /seahorn/build
+WORKDIR /seahorn/build
+
+ARG BUILD_TYPE
+
+# Build configuration.
+RUN cmake -GNinja \
+          -DCMAKE_BUILD_TYPE=$BUILD_TYPE \ 
+          -DBOOST_ROOT=/deps/boost \
+          -DZ3_ROOT=/deps/z3 \
+          -DLLVM_DIR=/deps/LLVM-3.8.1-Linux/share/llvm/cmake \
+          -DCMAKE_INSTALL_PREFIX=run \
+          -DCMAKE_CXX_COMPILER=g++-5 \
+          -DCPACK_GENERATOR="TGZ" \
+          -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \
+          ../ && \
+    cmake --build . --target extra  && cmake .. && \
+    cmake --build . --target crab  && cmake .. && \
+    cmake --build . --target install && \
+    cmake --build . --target package && \
+    # symlink clang (from base image)
+    ln -s /clang-3.8/bin/clang run/bin/clang && \
+    ln -s /clang-3.8/bin/clang++ run/bin/clang++
+
+ENV PATH "/seahorn/build/run/bin:$PATH"
+
+WORKDIR /seahorn
+
+RUN echo '#!/bin/sh' > /tmp/cpy-tarball.sh && \ 
+    echo 'cp /seahorn/build/*.tar.gz /host/ && bash' >> /tmp/cpy-tarball.sh && \
+    chmod +x /tmp/cpy-tarball.sh
+
+# Define default command.
+CMD ["/tmp/cpy-tarball.sh"]
```

Comment:
I don't think we need this anymore. In the current travis configuration, the image is copied out using `docker run` instead of as a side-effect of `docker-build`

Reply:
I was wondering if we still want to be able to build a tarball onto the host machine with Docker. In that case, it's convenient to set the default command here so that users only need to `docker run` the image after building.